packages/open-payments/src/client/payment-pointer.test.ts

```diff
@@ -0,0 +1,31 @@
+import { createPaymentPointerRoutes } from './payment-pointer'
+import { OpenAPI, HttpMethod, createOpenAPI } from 'openapi'
+import config from '../config'
+import { defaultAxiosInstance, silentLogger } from '../test/helpers'
+
+describe('payment-pointer', (): void => {
+  let openApi: OpenAPI
+
+  beforeAll(async () => {
+    openApi = await createOpenAPI(config.OPEN_PAYMENTS_OPEN_API_URL)
+  })
+
+  const axiosInstance = defaultAxiosInstance
+  const logger = silentLogger
```

Comment:
Just as an aside: I think once this package gets bigger we could shift to some kind of TestFactory generation for these mocks/defaults, but at this stage I don't see the need